PubMed XML.js

```diff
@@ -180,7 +180,12 @@ function doImport() {
 					var year = ZU.xpathText(pubDate, 'Year');
 
 					if (day) {
-						newItem.date = month + " " + day + ", " + year;
+						if (month && /\d+/.test(month)) {
```

Comment:
Could we just add a comment showing two examples of month values this is expecting?